pkgs/applications/version-management/nbstripout/default.nix

```diff
@@ -9,7 +9,7 @@ buildPythonApplication rec {
   # Mercurial should be added as a build input but because it's a Python
   # application, it would mess up the Python environment. Thus, don't add it
   # here, instead add it to PATH when running unit tests
-  buildInputs = [ pytest pytest-flake8 pytest-cram git pytestrunner ];
+  checkInputs = [ pytest pytest-flake8 pytest-cram git pytestrunner ];
```

Comment:
Is `pytestrunner` a `setup_requires`? If so it should be native.

Reply:
It is, but when I tried to remove it, I got ```No local packages or working download links found for pytest-runner```.

Reply:
Remove it, or moved it to `nativeBuildInputs`?

Reply:
Oh, I see now. Sorry :)